mantidproject/mantid
# Fix restore to default rescaling

**Description of work.**

``View --> Restore Default Layout`` now preserves the main window size 

**To test:**

1. Run workbench
2. Resize window
3. ``View --> Restore Default Layout``
4. The window size should be preserved.

Fixes #25795
To be merged after PR #25934


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
